### ISSUE TITLE
[Bug/Feature]: Controlling conference title abbreviation

### ISSUE THREAD

**Issue by juhaszp95** [retorquere/zotero-better-bibtex#2433]
### Debug log ID

MQ2Z5GJK-refs-euc

### What happened?

Hello,

I'm trying to export a conference paper, in which the conference title is given (see debug log MQ2Z5GJK-refs-euc). However, the name of the conference gets abbreviated (as the "abbreviate journal titles" setting is ON in the settings), which I would like to avoid. I would like BBT to use the abbreviated journal titles (which I can specify explicitly, which is good), but at the same time I would not like it to abbreviate the conference title (or at least I should have control over how it does it). If I disable journal title abbreviation, it disables abbreviation altogether (even though I want to keep it for journals).

Would it be possible to introduce the same "abbreviated conference title" field, like the one available for journal-type entries, for more granular control?

**Comment by retorquere** [comment]
Can you paste here what you want `MQ2Z5GJK-refs-euc` to look like when exported?

**Comment by juhaszp95** [comment]
I don't want the conference title to be abbreviated at all (while keeping the abbreviated journal titles). This could most flexibly be achieved by having an "abbreviated conference title" field available in Zotero, as is the "abbreviated journal field" for articles I'd already available (and which is very convenient).

**Comment by retorquere** [comment]
> I don't want the conference title to be abbreviated at all (while keeping the abbreviated journal titles). This could most flexibly be achieved by having an "abbreviated conference title" field available in Zotero, as is the "abbreviated journal field" field for articles is already available (and which is very convenient).

Can you paste an actual sample of how you want the output to be like? So not a description of the output, but the actual output?

**Comment by juhaszp95** [comment]
I've just solved my problem actually, by changing the "automatically abbreviate journal title if none is set explicitly" to OFF. This is debug lod ID 28469GJV-refs-euc just in case. By trial-and-error, I've realised that this applies not only to journal titles, but to proceedings titles too.

As per your question, in my entry the proceedings title is "Proceedings of the Second Workshop on the LLVM Compiler Infrastructure in HPC" and I wanted it to come out exactly this way (i.e. no abbreviations), while keeping abbreviations in the journal titles.
